[2025-05-14T15:45:14.248+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: Sales_Performance_ETL.Load_Data scheduled__2025-05-14T15:40:00+00:00 [queued]>
[2025-05-14T15:45:14.261+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: Sales_Performance_ETL.Load_Data scheduled__2025-05-14T15:40:00+00:00 [queued]>
[2025-05-14T15:45:14.261+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2025-05-14T15:45:14.263+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 2
[2025-05-14T15:45:14.263+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2025-05-14T15:45:14.279+0000] {taskinstance.py:1389} INFO - Executing <Task(BashOperator): Load_Data> on 2025-05-14 15:40:00+00:00
[2025-05-14T15:45:14.285+0000] {standard_task_runner.py:52} INFO - Started process 28634 to run task
[2025-05-14T15:45:14.287+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'Sales_Performance_ETL', 'Load_Data', 'scheduled__2025-05-14T15:40:00+00:00', '--job-id', '125', '--raw', '--subdir', 'DAGS_FOLDER/P2M3_Hisham_Audiputra_Suharto_DAG.py', '--cfg-path', '/tmp/tmpq0p4c_3s', '--error-file', '/tmp/tmprjea3li5']
[2025-05-14T15:45:14.289+0000] {standard_task_runner.py:80} INFO - Job 125: Subtask Load_Data
[2025-05-14T15:45:14.359+0000] {task_command.py:371} INFO - Running <TaskInstance: Sales_Performance_ETL.Load_Data scheduled__2025-05-14T15:40:00+00:00 [running]> on host 2ca7303636ab
[2025-05-14T15:45:14.437+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Hisham
AIRFLOW_CTX_DAG_ID=Sales_Performance_ETL
AIRFLOW_CTX_TASK_ID=Load_Data
AIRFLOW_CTX_EXECUTION_DATE=2025-05-14T15:40:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2025-05-14T15:40:00+00:00
[2025-05-14T15:45:14.438+0000] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2025-05-14T15:45:14.440+0000] {subprocess.py:74} INFO - Running command: ['/bin/bash', '-c', 'sudo -u *** python /opt/***/scripts/Load_Data.py']
[2025-05-14T15:45:14.450+0000] {subprocess.py:85} INFO - Output:
[2025-05-14T15:45:16.134+0000] {subprocess.py:92} INFO - WARNING: An illegal reflective access operation has occurred
[2025-05-14T15:45:16.136+0000] {subprocess.py:92} INFO - WARNING: Illegal reflective access by org.apache.spark.unsafe.Platform (file:/home/***/.local/lib/python3.7/site-packages/pyspark/jars/spark-unsafe_2.12-3.1.2.jar) to constructor java.nio.DirectByteBuffer(long,int)
[2025-05-14T15:45:16.137+0000] {subprocess.py:92} INFO - WARNING: Please consider reporting this to the maintainers of org.apache.spark.unsafe.Platform
[2025-05-14T15:45:16.137+0000] {subprocess.py:92} INFO - WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
[2025-05-14T15:45:16.138+0000] {subprocess.py:92} INFO - WARNING: All illegal access operations will be denied in a future release
[2025-05-14T15:45:16.284+0000] {subprocess.py:92} INFO - :: loading settings :: url = jar:file:/home/***/.local/lib/python3.7/site-packages/pyspark/jars/ivy-2.4.0.jar!/org/apache/ivy/core/settings/ivysettings.xml
[2025-05-14T15:45:16.355+0000] {subprocess.py:92} INFO - Ivy Default Cache set to: /home/***/.ivy2/cache
[2025-05-14T15:45:16.356+0000] {subprocess.py:92} INFO - The jars for the packages stored in: /home/***/.ivy2/jars
[2025-05-14T15:45:16.359+0000] {subprocess.py:92} INFO - org.postgresql#postgresql added as a dependency
[2025-05-14T15:45:16.360+0000] {subprocess.py:92} INFO - :: resolving dependencies :: org.apache.spark#spark-submit-parent-7352ffc7-346b-4bac-828e-74df55f83334;1.0
[2025-05-14T15:45:16.361+0000] {subprocess.py:92} INFO - 	confs: [default]
[2025-05-14T15:45:16.448+0000] {subprocess.py:92} INFO - 	found org.postgresql#postgresql;42.6.0 in central
[2025-05-14T15:45:16.470+0000] {subprocess.py:92} INFO - 	found org.checkerframework#checker-qual;3.31.0 in central
[2025-05-14T15:45:16.489+0000] {subprocess.py:92} INFO - :: resolution report :: resolve 120ms :: artifacts dl 9ms
[2025-05-14T15:45:16.490+0000] {subprocess.py:92} INFO - 	:: modules in use:
[2025-05-14T15:45:16.491+0000] {subprocess.py:92} INFO - 	org.checkerframework#checker-qual;3.31.0 from central in [default]
[2025-05-14T15:45:16.491+0000] {subprocess.py:92} INFO - 	org.postgresql#postgresql;42.6.0 from central in [default]
[2025-05-14T15:45:16.492+0000] {subprocess.py:92} INFO - 	---------------------------------------------------------------------
[2025-05-14T15:45:16.492+0000] {subprocess.py:92} INFO - 	|                  |            modules            ||   artifacts   |
[2025-05-14T15:45:16.493+0000] {subprocess.py:92} INFO - 	|       conf       | number| search|dwnlded|evicted|| number|dwnlded|
[2025-05-14T15:45:16.493+0000] {subprocess.py:92} INFO - 	---------------------------------------------------------------------
[2025-05-14T15:45:16.494+0000] {subprocess.py:92} INFO - 	|      default     |   2   |   0   |   0   |   0   ||   2   |   0   |
[2025-05-14T15:45:16.495+0000] {subprocess.py:92} INFO - 	---------------------------------------------------------------------
[2025-05-14T15:45:16.496+0000] {subprocess.py:92} INFO - :: retrieving :: org.apache.spark#spark-submit-parent-7352ffc7-346b-4bac-828e-74df55f83334
[2025-05-14T15:45:16.497+0000] {subprocess.py:92} INFO - 	confs: [default]
[2025-05-14T15:45:16.503+0000] {subprocess.py:92} INFO - 	0 artifacts copied, 2 already retrieved (0kB/7ms)
[2025-05-14T15:45:16.691+0000] {subprocess.py:92} INFO - 25/05/14 15:45:16 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2025-05-14T15:45:16.918+0000] {subprocess.py:92} INFO - Using Spark's default log4j profile: org/apache/spark/log4j-defaults.properties
[2025-05-14T15:45:16.919+0000] {subprocess.py:92} INFO - Setting default log level to "WARN".
[2025-05-14T15:45:16.919+0000] {subprocess.py:92} INFO - To adjust logging level use sc.setLogLevel(newLevel). For SparkR, use setLogLevel(newLevel).
[2025-05-14T15:45:27.876+0000] {subprocess.py:96} INFO - Command exited with return code 0
[2025-05-14T15:45:27.927+0000] {taskinstance.py:1412} INFO - Marking task as SUCCESS. dag_id=Sales_Performance_ETL, task_id=Load_Data, execution_date=20250514T154000, start_date=20250514T154514, end_date=20250514T154527
[2025-05-14T15:45:27.992+0000] {local_task_job.py:156} INFO - Task exited with return code 0
[2025-05-14T15:45:28.039+0000] {local_task_job.py:279} INFO - 0 downstream tasks scheduled from follow-on schedule check
